apps/preview/next/pages/showcases/Banners/Hero.tsx

```diff
@@ -0,0 +1,41 @@
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { SfButton } from '@storefront-ui/react';
+import headphones from '@assets/hero-headphones.png';
+import mobileBackground from '@assets/hero-bg-mobile.png';
+import desktopBackground from '@assets/hero-bg.png';
+
+const variables = {
+  '--sf-img-mobile': `url(${mobileBackground.src})`,
+  '--sf-img-desktop': `url(${desktopBackground.src})`,
+} as React.CSSProperties;
+
+export default function Hero() {
+  return (
+    <div
+      style={variables}
+      className="md:flex md:flex-row-reverse min-h-[600px] max-w-[1536px] mx-auto bg-[url:var(--sf-img-mobile)] md:bg-[url:var(--sf-img-desktop)] bg-center bg-cover bg-no-repeat"
+    >
+      <div className="flex flex-col md:basis-2/4 md:items-stretch md:overflow-hidden">
+        <img src={headphones.src} alt="Headphones alt" className="h-full object-cover object-left" />
```

Comment:
alt could be without `alt`? :) 
```suggestion
        <img src={headphones.src} alt="Headphones" className="h-full object-cover object-left" />
```

Reply:
ok